packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -231,10 +239,12 @@ pub trait Interval<T>: Sized {
             let (lhs_lower, lhs_upper) = self.into_bound();
             let (rhs_lower, rhs_upper) = other.into_bound();
 
-            // The ranges overlaps
-            // Range A:   [-----] | [-----]
-            // Range B: [-----]   |   [-----]
-            // Result:    [---]   |   [---]
+            // Examples  |      1    |     2
+            // ==========|===========|===========
+            // Range A   |   [-----] | [-----]
+            // Range B   | [-----]   |   [-----]
+            // ----------|-----------|-----------
+            // Intersect |   [---]   |   [---]
```

Comment:
I don't feel strongly about this, I know the method is called `intersect`, just reading the table felt a bit strange.

```suggestion
            // Examples     |     1     |     2
            // =============|===========|===========
            // Range A      |   [-----] | [-----]
            // Range B      | [-----]   |   [-----]
            // -------------|-----------|-----------
            // Intersection |   [---]   |   [---]
```

Alternatively, could just fix the heading for `1`
```suggestion
            // Examples  |     1     |     2
            // ==========|===========|===========
            // Range A   |   [-----] | [-----]
            // Range B   | [-----]   |   [-----]
            // ----------|-----------|-----------
            // Intersect |   [---]   |   [---]
```